machine-learning/ml/preprocessing/preprocessing.py

```diff
@@ -70,7 +75,7 @@ def retrieve_labelled_instances(dataset, refactoring: LowLevelRefactoring):
 
     # do we want to try the models without process and authorship metrics?
     if DROP_PROCESS_AND_AUTHORSHIP_METRICS:
-        x = x.drop(["authorOwnership", "bugFixCount", "linesAdded", "linesDeleted", "qtyMajorAuthors",
+        x = x.drop(["authorOwnership", "bugFixCount", "qtyMajorAuthors",
```

Comment:
we decided to drop them directly from the db, right? great!

Reply:
Yes